doc/changelog/pendings/changes_1624799411242887600.md

```diff
@@ -0,0 +1,13 @@
+Changelog for commit "[fix(Core/Player): Use SkillLineAbility.dbc to determine player initial spells - skill assignment done in a new table `playercreateinfo_skills`](https://github.com/azerothcore/azerothcore-wotlk/commit/1be561e03b56dc396270335886e59eddad9fa0c6)"
+
+### Added
+
+- playercreateinfo_skills - New Database table for skill assignment.
+
+### Removed
+
+- playercreateinfo_spells
+
+### Changed
+
+- Use SkillLineAbility.dbc to determine player initial spells.
```

Comment:
An #how to upgrade section here is needed because that skilline class is public and can be used by some modules.

Reply:
Done